Revise partial bundle section in README (#5702)
plotly/plotly.js: README.md

#### Read the [Getting started page](https://plotly.com/javascript/getting-started/) for more examples.

## Partial bundles

There are two kinds of plotly.js partial bundles:

> 1. The official partial bundles that are distributed to `npm` and `CDN` covered by [the dist README](https://github.com/plotly/plotly.js/blob/master/dist/README.md).
> 2. The custom ones that could be created by the user, if none of the distributed packages meet your needs.

Reviewer: ```suggestion
1. The official partial bundles that are distributed to `npm` and the CDN, described in [the dist README](https://github.com/plotly/plotly.js/blob/master/dist/README.md).
2. Custom bundles you can create yourself, if none of the distributed packages meet your needs.
```

> Use `traces` option to create a custom bundle of desired traces.

Reviewer: ```suggestion
Use the `traces` option to include just the trace types you need.
```

```
npm run partial-bundle -- --traces scatter,scattergl,scatter3d
```

> Please note that `scatter` trace is currently added to the bundle by default.
> [Whereas this behaviour may be subject to change in future](https://github.com/plotly/plotly.js/pull/5535), we recommend that you explicitly include `scatter` trace in your scripts in case `scatter` trace is needed in your bundle.

Reviewer: ```suggestion
Please note that the `scatter` trace is currently included in all bundles and cannot be removed.
[This behaviour may change in the future](https://github.com/plotly/plotly.js/pull/5535), so we recommend that you explicitly include `scatter` anyway if you need it in your bundle.
```

> If not specified otherwise all transforms are included in the bundle.
> Use `transforms` option to specify which transforms should be included.

Reviewer: ```suggestion
By default all transforms are included in the bundle.
Use the `transforms` option to specify which should be included.
```

```
npm run partial-bundle -- --transforms sort,filter
```

> Use `transforms none` option to exclude all transforms.

Reviewer: ```suggestion
Or use `transforms none` to exclude them all.
```

```
npm run partial-bundle -- --transforms none
```

> Use `out` option to change default `custom` bundle name.

Reviewer: ```suggestion
Use the `out` option to change the bundle filename (default `custom`).
The new bundle will be created in the `dist/` directory and named `plotly-<out>.min.js` or `plotly-<out>.js` if unminified.
```

```
npm run partial-bundle -- --out myBundleName
```

> Use `unminified` option to create an `unminified` bundle.

Reviewer: ```suggestion
Use the `unminified` option to disable compression.
```

```
npm run partial-bundle -- --unminified
```

### Example illustrating use of different options together

> To create `unminified` custom bundle named `myScatters` including `scatter`, `scattergl` and `scatter3d` traces without any transforms:

Reviewer: ```suggestion
To create an unminified custom bundle named `myScatters` including `scatter`, `scattergl` and `scatter3d` traces without any transforms:
```
